docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -20,11 +20,11 @@ helpviewer_keywords:
 ---
 # Standard numeric format strings
 
-Standard numeric format strings are used to format common numeric types. A standard numeric format string takes the form `Axx`, where:
+Standard numeric format strings are used to format common numeric types. A standard numeric format string takes the form `Axxx`, where:
 
 - `A` is a single alphabetic character called the *format specifier*. Any numeric format string that contains more than one alphabetic character, including white space, is interpreted as a custom numeric format string. For more information, see [Custom Numeric Format Strings](custom-numeric-format-strings.md).
 
-- `xx` is an optional integer called the *precision specifier*. The precision specifier ranges from 0 to 99 and affects the number of digits in the result. Note that the precision specifier controls the number of digits in the string representation of a number. It does not round the number itself. To perform a rounding operation, use the <xref:System.Math.Ceiling%2A?displayProperty=nameWithType>, <xref:System.Math.Floor%2A?displayProperty=nameWithType>, or <xref:System.Math.Round%2A?displayProperty=nameWithType> method.
+- `xxx` is an optional integer called the *precision specifier*. The precision specifier affects the number of digits in the resulting string. In .NET 6 and later versions, the maximum precision value is <xref:System.Int32.MaxValue?displayProperty=nameWithType>. In previous .NET versions, the precision can range from 0 to 99. The precision specifier controls the number of digits in the string representation of a number. It does not round the number itself. To perform a rounding operation, use the <xref:System.Math.Ceiling%2A?displayProperty=nameWithType>, <xref:System.Math.Floor%2A?displayProperty=nameWithType>, or <xref:System.Math.Round%2A?displayProperty=nameWithType> method.
```

Comment:
Not sure if `xxx` is the best way to show the placeholder for the precision since it can be much more than 3 digits. Open to suggestions.

Reply:
Since these are placeholders, what about `A###`

Reply:
# might get confused with the digit placeholder `#` of a custom numeric format string: https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings